client/state/partner-portal/licenses/handlers.ts

```diff
@@ -50,7 +50,7 @@ interface APIItemFormatter< FormattedType, APIType > {
 	( items: APIType[] ): FormattedType[];
 }
 
-function http( options, action: HttpAction ): AnyAction {
+function http( options: any, action: HttpAction ): AnyAction {
```

Comment:
Normally I wouldn't use `any` but in this case it's actually required because that's the type of the argument to `coreHttp`.